blog/2024-12-05-boa-release-020/index.mdx

```diff
@@ -0,0 +1,199 @@
+you can even use it from WebAssembly. See the [about](/about) page for more info.
+
+In this release, our conformance has grown from 87.3% to 87.5% in the official ECMAScript Test Suite
+(Test262). This small jump is expected as we're shifting most of our focus to performance as the majority of the engine is now conformant. We will continue to implement more of the specification as we go along but we expect these changes to be much smaller than we've been used to.
+
+You can check the full list of changes [here][changelog], and the full information on conformance
+[here][conformance].
+
+<!--truncate-->
+
+## Feature Highlights
+
+### Temporal
+
+Boa is continuing to progress on [Temporal](https://github.com/tc39/proposal-temporal). The Temporal API is a new
+set of built-in objects and functions that is designed to be a more modern replacement for the `Date`
+object, providing a more feature-rich and flexible API for working with dates and times.
+
+It is currently a [stage 3 proposal](https://tc39.es/proposal-temporal/docs/) and we are working
+alongside the TC39 champions to put together a solid Rust implementation. Since Temporal is such an
+extensive specification, we have done most of the work outside of Boa so that it can be used in other
+projects. This work can be found in the [temporal_rs](https://github.com/boa-dev/temporal/) repository.
+
+We hope to release a full blog post on Temporal in the future, but for now you can see the previous release notes for some examples on how to use it.
+You can also look at the [Temporal Cook Book](https://tc39.es/proposal-temporal/docs/cookbook.html) for some examples too!
+
+If you're interested in learning more or want to contribute to the native Rust implementation of
+Temporal, feel free to check out `temporal_rs`'s [issues](https://github.com/boa-dev/temporal/issues)!
+
+Boa's conformance on the Temporal test suite has grown from 24.61% to 40.67% in this release.
+
+### Nightlies
+
+Boa now supports nightly releases, this was originally created to aid with the testing of conformance for test262.fyi. This is a great way to see the latest changes and help offer feedback on new features or just to see the latest changes in the engine. You can find the nightly releases [here](https://github.com/boa-dev/boa/releases/tag/nightly)
+
+### Atomics.pause
+
+Boa has added support for the [stage 3 proposal `Atomics.pause`](https://github.com/tc39/proposal-atomics-microwait). This function is used to pause the execution of a thread for a specified amount of time. This function is useful for implementing spinlocks and other synchronization primitives in JavaScript.
+
+### Getters and Setters in the js_class! macro
+
+You can now add getters and setters to the `js_class!` macro. This allows you to define getters and setters on your JavaScript classes in Rust. This is a feature that has been requested by many users of Boa, and thanks to @hansl we now have it!
+[Point to example]
```

Comment:
Added a possible example.

```suggestion
```rust
#[derive(Clone, Default, Trace, Finalize, JsData)]
pub struct Game {
    score: u32,
}

js_class! {
    class Game {
        property score {
            get(this: JsClass<Game>) -> u32 {
                this.borrow().score
            }

            set(this: JsClass<Game>, new_value: u32) {
                this.borrow_mut().score = new_value;
            }
        }

        constructor() {
            Ok(Game::default())
        }
    }
}
```
```